test/rules/object-literal-sort-keys/allow-newline-grouping/test.ts.lint

```diff
@@ -0,0 +1,41 @@
+var passA = {
+    a: 1,
+    b: 2,
+    d: 4,
+
+    c: 3,
+    e: 5,
+};
```

Comment:
please add one of these test cases to an existing test to check if the new behavior is not enabled without the option